engine/class_modules/apl/ConvertAPL.py

```diff
@@ -119,41 +113,28 @@ def main(argv):
                 print('Invalid spec selected')
                 sys.exit(2)
     subaplList, aplList = read_apl(inputFilePath)
-    try:
-        outputFile = open(outputFilePath, 'r')
-    except IOError as e:
-        print('Error opening output file with path: ' + outputFilePath)
-        print(e)
-        sys.exit(2)
-    else:
-        try:
-            tempFile = open('tempFile.txt', 'w')
```

Comment:
The Python stdlib has a dedicated `tempfile` module that would be more suitable here; leaving to a separate PR